tests/generation/Bears.py

```diff
@@ -357,3 +357,17 @@ def test_filter_bears_ci_mode(self):
                                     if bear not in res_1[lang]]
                 for bear in additional_bears_by_lang[lang]:
                     self.assertIn(bear, additional_bears)
+
+    def test_print_coala_quickstart_version_short(self):
+        sys.argv.append('-v')
+        with retrieve_stdout() as custom_stdout:
+            with self.assertRaises(SystemExit) as cm:
+                main()
+            self.assertEquals('0.4.0', custom_stdout.getvalue().strip('\n'))
```

Comment:
optional: why not just testing against `0.4.0\n` and saving the `strip`? ;)

Reply:
Nice, I will amend it